src/commands/config.py

```
@@ -187,5 +237,173 @@ async def set_games(interaction: discord.Interaction, games: str):
+    if guild_id is None:
+        await interaction.followup.send(
+            "This command must be run in a server.", ephemeral=True
+        )
+        return
+
+    try:
+        enabled_games = await _load_enabled_games(guild_id)
+        normalized = _normalize_games_list([*enabled_games, game])
+    except ValueError:
+        await interaction.followup.send(
+            "Unsupported game. Choose from: " f"{_supported_games_text()}",
```

Ruff (0.15.5)

[warning] Implicitly concatenated string literals on one line

(ISC001)